core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -211,18 +211,13 @@ private Path download(Terminal terminal, String pluginId, Path tmpDir) throws Ex
             final String url;
             final String stagingHash = System.getProperty(PROPERTY_STAGING_ID);
             if (stagingHash != null) {
-                url = String.format(
-                        Locale.ROOT,
-                        "https://download.elastic.co/elasticsearch/staging/%1$s-%2$s/org/elasticsearch/plugin/%3$s/%1$s/%3$s-%1$s.zip",
-                        version,
-                        stagingHash,
-                        pluginId);
+                url = String.format(Locale.ROOT,
+                    "https://staging.elastic.co/%1$s/download/elasticsearch-plugins/%2$s/%2$s-%3$s.zip",
+                    stagingHash, pluginId, version);
             } else {
-                url = String.format(
-                        Locale.ROOT,
-                        "https://download.elastic.co/elasticsearch/release/org/elasticsearch/plugin/%1$s/%2$s/%1$s-%2$s.zip",
-                        pluginId,
-                        version);
+                url = String.format(Locale.ROOT,
+                    "https://artifacts.elastic.co/download/elasticsearch-plugins/%1$s/%1$s-%2$s.zip",
```

Comment:
Then that is still a reason to talk about the merits of continuing to publish plugin zips to maven. But I do not think that should affect the location of our plugins that will be linked by the website for offline installation.  Note that we still have maven artifacts with the unified release (for published artifacts, that is `https://artifacts.elastic.co/maven`).